highfidelity/hifi
# WL 21422 - Add a scale slider for avatar size

# Worklist 21422
> In Settings >>> Avatar, add a slider to the Avatar scale field. Min ~ 0.01, Middle ~ 1.0, Right ~ 3.0. Users should be able to use either the slider or the text input to resize their avatar.
> -The value range should be not be given in exact scale multiples because domains can set their own min/max
> -There should be a "Reset scale button next to the slider to give users a quick one click way to get back to normal
> per Worklist https://worklist.net/21422

## Test Plan
1. Download and run this PR
2. Open up Avatar Settings (_Settings > Avatar.._)
3. Confirm that you see the Avatar Scale setting, and that it has a Slider, an Input and a 'Reset' button.
4. Confirm that when changing either the slider or the input, that the other syncs up.
5. Confirm that changing both settings can Save Changes.
6. Confirm that hitting the 'Reset' button, either resets the value to 1 if the Domain Max is 1 or above - or it resets it to the Max the domain can handle if below 1.0.
7. Run `MyAvatar.getDomainMinScale()` in console, confirm that it prints your current domain's avatar minimum scale limit.
8. Run `MyAvatar.getDomainMaxScale()` in console, confirm that it prints your current domain's avatar maximum scale limit.
9. Confirm that every other Scrollbar in the menu's is looking and functioning okay.
10. Retest Steps 2 - 6 and confirm that they all work as expected on the Tablet in HMD.
